SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -240,9 +240,14 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 	// runtime.exe --config=squid_config.json --snapshot=snapshots\default.snapshot --worker-port 8018 --http-port 5006 --grpc-port 7777
 	// --worker-external-host 127.0.0.1 --snapshots-directory=spatial/snapshots.
 	FString RuntimeArgs = FString::Printf(TEXT("--config=\"%s\" --snapshot=\"%s\" --worker-port %s --http-port %s --grpc-port %s "
-											   "--worker-external-host %s --snapshots-directory=\"%s\" %s"),
+											   "--snapshots-directory=\"%s\" %s"),
 										  *LaunchConfig, *SnapshotName, *FString::FromInt(WorkerPort), *FString::FromInt(HTTPPort),
-										  *FString::FromInt(GRPCPort), *RuntimeIPToExpose, *SnapshotPath, *LaunchArgs);
+										  *FString::FromInt(GRPCPort), *SnapshotPath, *LaunchArgs);
+
+	if (!RuntimeIPToExpose.IsEmpty())
+	{
+		RuntimeArgs.Append(FString::Printf(TEXT(" --worker-external-host %s"), *RuntimeIPToExpose));
```

Comment:
Another fix for earlier mistakes. The snapshot-directory was not getting parsed correctly if we didn't have a RuntimeIPToExpose set.